pkg/subctl/cmd/deploybroker.go

```diff
@@ -24,16 +24,28 @@ import (
 	"github.com/submariner-io/submariner-operator/pkg/broker"
 	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+	lighthouse "github.com/submariner-io/submariner-operator/pkg/subctl/lighthouse/deploy"
 )
 
-var enableDataplane bool
-var disableDataplane bool
+var (
+	enableDataplane              bool
+	disableDataplane             bool
+	serviceDiscovery             bool
+	serviceDiscoveryImageRepo    string
+	serviceDiscoveryImageVersion string
+)
 
 func init() {
 	deployBroker.PersistentFlags().BoolVar(&enableDataplane, "dataplane", false,
 		"Install the Submariner dataplane on the broker")
 	deployBroker.PersistentFlags().BoolVar(&disableDataplane, "no-dataplane", true,
 		"Don't install the Submariner dataplane on the broker (default)")
+	deployBroker.PersistentFlags().BoolVar(&serviceDiscovery, "service-discovery", false,
+		"Enable Multi Cluster Service Discovery")
+	deployBroker.PersistentFlags().StringVar(&serviceDiscoveryImageRepo, "service-discovery-repo", "",
```

Comment:
We’ll have to specify defaults here, but that can wait until we have the images we want in the first release.

Reply:
seems ok as a start, we can model the flags more later when we have a better idea of the experience, and eventually this can be disable-service-discovery ;)